From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-list_lru-replace-linear-array-with-xarray-fix

remove unused local

mm/list_lru.c: In function 'memcg_list_lru_alloc':
mm/list_lru.c:477:32: error: unused variable 'mlrus' [-Werror=unused-variable]
  477 |         struct list_lru_memcg *mlrus;
      |                                ^~~~~

Cc: Muchun Song <songmuchun@bytedance.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/list_lru.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/list_lru.c~mm-list_lru-replace-linear-array-with-xarray-fix
+++ a/mm/list_lru.c
@@ -480,7 +480,6 @@ int memcg_list_lru_alloc(struct mem_cgro
 {
 	int i;
 	unsigned long flags;
-	struct list_lru_memcg *mlrus;
 	struct list_lru_memcg_table {
 		struct list_lru_per_memcg *mlru;
 		struct mem_cgroup *memcg;
_
